bokehjs/gulp/tasks/scripts.coffee

```diff
@@ -18,22 +19,25 @@ gulp.task "scripts", ->
       .transform "coffeeify"
       .bundle()
 
-  gulp.src paths.coffee.sources
+  return gulp.src paths.coffee.sources
     .pipe browserified
     .pipe change (content) ->
       "(function() { var define = undefined; #{content} })()"
     .pipe rename "bokeh.js"
     .pipe gulp.dest paths.buildDir.js
 
-minifyTask = ->
-  gulp.src paths.coffee.destination.fullWithPath
+gulp.task "scripts:minify", ->
+  return gulp.src paths.coffee.destination.fullWithPath
```

Comment:
Style note -- this has mixed explicit / implicit returns.  My preference for the file is implicit, but it should be one style and used throughout.